src/testers.jl

```diff
@@ -28,25 +28,28 @@ Call `FiniteDifferences.j′vp`, with the option to ignore certain `xs`.
 - `∂xs::Tuple`: Derivatives estimated by finite differencing.
 """
 function _make_fdm_call(fdm, f, ȳ, xs, ignores)
-    sig = Expr(:tuple)
-    call = Expr(:call, f)
-    newxs = Any[]
-    arginds = Int[]
-    i = 1
-    for (x, ignore) in zip(xs, ignores)
-        if ignore
-            push!(call.args, x)
-        else
-            push!(call.args, Symbol(:x, i))
-            push!(sig.args, Symbol(:x, i))
-            push!(newxs, x)
-            push!(arginds, i)
+    ignores = collect(ignores)
+    function f2(sigargs...)
+        callargs = Any[]
+        j = 1
+        for (i, (x, ignore)) in enumerate(zip(xs, ignores))
```

Comment:
```suggestion

        for (i, (x, ignore)) in enumerate(zip(xs, ignores))
```

Reply:
can we drop the `enumerate` here? is `i` used anywhere?